assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -73,10 +74,12 @@ const Location = ({ vehicle }: { vehicle: Vehicle }) => {
   const routeVehicles: Vehicle[] = useRouteVehicles(vehicle.routeId, vehicle.id)
   const shapes: Shape[] = useTripShape(vehicle.tripId)
   const { isOffCourse, latitude, longitude, stopStatus } = vehicle
-  const nearestIntersection: string | null = useNearestIntersection(
-    latitude,
-    longitude
-  )
+
+  const nearestIntersectionResult = useNearestIntersection(latitude, longitude)
+  const nearestIntersection = isOk(nearestIntersectionResult)
+    ? nearestIntersectionResult.ok
+    : null
```

Comment:
note: when working on this idea before, I was struggling deciding whether to implement `Result` myself or use a library, one of the benifits of using a library is that this kind of transformation(and others) would be provided by default via `unwrapOr` if we used something like https://github.com/vultix/ts-results

I'm unsure what the best "Rust `Result<T,E>` type but in Typescript" Library is, and there's a few, but it's also not necessary, just QOL.